### PR TITLE
Indibranch

### PR DESCRIPTION
**Pull request by indigophillips**
Added bootstrap file system which includes: js,styles,img,lib, contact form and index.hml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,892 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Pohutukawa</title>
+      <meta content="width=device-width, initial-scale=1.0" name="viewport">
+      <meta content="" name="keywords">
+      <meta content="" name="description">
+    
+      <!-- Favicons -->
+      <link href="../pohutukawa.io/img/faviconpohutu.png" rel="icon">
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove **_../pohutukawa.io/_**

Should be this: &lt;link href=&quot;img/faviconpohutu.png&quot; rel=&quot;icon&quot;&gt;

File: index.html [hunk 2]
```diff
@@ -0,0 +1,892 @@
...
+      <section id="intro">
+        <div class="intro-container">
+          <div id="introCarousel" class="carousel  slide carousel-fade" data-ride="carousel">
+    
+            <ol class="carousel-indicators"></ol>
+    
+            <div class="carousel-inner" role="listbox">
+    
+              <div class="carousel-item active">
+                <div class="carousel-background"><img src="../pohutukawa.io/img/intro-carousel/1.jpeg" alt=""></div>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

Should be this: &lt;img src=&quot;img/intro-carousel/1.jpeg&quot; alt=&quot;&quot;&gt;

File: index.html [hunk 3]
```diff
@@ -0,0 +1,892 @@
...
+                  <div class="carousel-content">
+                    <h2>We are mātauranga Māori</h2>
+                    <p>(noun) Māori knowledge - the body of knowledge originating from Māori ancestors, including the Māori world view and perspectives, Māori creativity and cultural practices.</p>
+                    <a href="#featured-services" class="btn-get-started scrollto">Get Started</a>
+                  </div>
+                </div>
+              </div>
+    
+              <div class="carousel-item">
+                <div class="carousel-background"><img src="../pohutukawa.io/img/intro-carousel/2.jpg" alt=""></div>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

Should be this: &lt;img src=&quot;img/intro-carousel/2.jpg&quot; alt=&quot;&quot;&gt;

File: index.html [hunk 4]
```diff
@@ -0,0 +1,892 @@
...
+                  <div class="carousel-content">
+                    <h2>We are mātanga hangarau</h2>
+                    <p>(noun) technologist, expert in technology. An information technology expert works to monitor and manage computer hardware, software and networks within an organization.</p>
+                    <a href="#featured-services" class="btn-get-started scrollto">Get Started</a>
+                  </div>
+                </div>
+              </div>
+    
+              <div class="carousel-item">
+                <div class="carousel-background"><img src="../pohutukawa.io/img/intro-carousel/3.jpeg" alt=""></div>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

Should be this: &lt;img src=&quot;img/intro-carousel/3.jpeg&quot; alt=&quot;&quot;&gt;

File: index.html [hunk 5]
```diff
@@ -0,0 +1,892 @@
...
+                  <div class="carousel-content">
+                    <h2>We are auaha</h2>
+                    <p>(adjective) having the quality of power to create.</p>
+                    <a href="#featured-services" class="btn-get-started scrollto">Get Started</a>
+                  </div>
+                </div>
+              </div>
+    
+              <div class="carousel-item">
+                <div class="carousel-background"><img src="../pohutukawa.io/img/intro-carousel/4.jpeg" alt=""></div>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

Should be this: &lt;img src=&quot;img/intro-carousel/4.jpeg&quot; alt=&quot;&quot;&gt;

File: index.html [hunk 6]
```diff
@@ -0,0 +1,892 @@
...
+                  <div class="carousel-content">
+                    <h2>We are haemata</h2>
+                    <p>(modifier) strong-growing, vigorous.</p>
+                    <a href="#featured-services" class="btn-get-started scrollto">Get Started</a>
+                  </div>
+                </div>
+              </div>
+    
+              <div class="carousel-item">
+                <div class="carousel-background"><img src="../pohutukawa.io/img/intro-carousel/5.jpeg" alt=""></div>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

File: index.html [hunk 7]
```diff
@@ -0,0 +1,892 @@
...
+            &copy; Copyright <strong>Pohutukawa</strong>. All Rights Reserved
+          </div>
+          <div class="credits">
+            <!--
+              All the links in the footer should remain intact.
+              You can delete the links only if you purchased the pro version.
+              Licensing information: https://bootstrapmade.com/license/
+              Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/buy/?theme=BizPage
+            -->
+            Designed by  <a href="../pohutukawa.io/img/whiteonblack.png" style="font-size:16px">Ātaahua Developments</a></p>
```
Comment (john-shortland) [inline index.html]: [Bug] Need to remove reference to the project folder on the local environment.

E.g. Remove ../pohutukawa.io/

File: index.html (hunk 1 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

File: index.html (hunk 6 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

File: index.html (hunk 5 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

File: index.html (hunk 1 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

File: index.html (hunk 3 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

File: index.html (hunk 1 above)
Comment (indigophillips) [reply index.html]: Deleted two instances of pohutukawa.io. As previous git push changes we made in later pushes yesterday.

**Comment by indigophillips** [comment]
Works via Mozilla, Google.

**Pull request merged**